src/PlusVolumeReconstruction/vtkPlusPasteSliceIntoVolume.cxx

```diff
@@ -212,6 +226,18 @@ vtkImageData* vtkPlusPasteSliceIntoVolume::GetAccumulationBuffer()
 }
 
 //----------------------------------------------------------------------------
+vtkImageData* vtkPlusPasteSliceIntoVolume::GetImportanceMask()
+{
+  return this->ImportanceMask;
+}
+
+//----------------------------------------------------------------------------
+void vtkPlusPasteSliceIntoVolume::SetImportanceMask(vtkImageData* importanceMask)
```

Comment:
If there's no functionality to the accessors, could you use the vtkSet/Get macros in the header?